lib/internal/repl.js

```diff
@@ -55,15 +55,26 @@ function createRepl(env, opts, cb) {
   }
 
   const repl = REPL.start(opts);
-  if (opts.terminal && env.NODE_REPL_HISTORY !== '') {
```

Comment:
I wonder if it wouldn't be more DRY if the logic was moved out to here. We can make `var historyPath = env.NODE_REPL_HISTORY` up here so we don't access the env programmatically multiple times.